.codeclimate.yml

```diff
@@ -84,6 +84,7 @@ exclude_paths:
   - 'lib/action_dispatch/rails5_ssl.rb'
   - "lib/developer_portal/app/views/developer_portal/javascripts/*"
   - "lib/developer_portal/app/views/developer_portal/css/*"
+  - "lib/developer_portal/app/assets/stylesheets/**/*"
```

Comment:
We should not have assets here

Reply:
Well we have this: https://github.com/3scale/porta/blob/master/lib/developer_portal/app/assets/stylesheets/plans_widget.scss
I just renamed from css to scss, but it was on that directory
Should I move it to `lib/developer_portal/app/views/developer_portal/css/` ?